shell/platform/android/platform_view_android_jni_impl.cc

```diff
@@ -1910,4 +1920,35 @@ bool PlatformViewAndroidJNIImpl::RequestDartDeferredLibrary(
   return true;
 }
 
+void PlatformViewAndroidJNIImpl::OnRasterStart(
+    const FrameTimingsRecorder& frame_timings_recorder) {
+  fml::TimePoint now = fml::TimePoint::Now();
+  fml::TimePoint wall_time = fml::TimePoint::CurrentWallTime();
+
+  JNIEnv* env = fml::jni::AttachCurrentThread();
+  auto java_object = java_object_.get(env);
+  if (java_object.is_null()) {
+    return;
+  }
+
+  fml::TimePoint build_start_wall_time =
```

Comment:
Why are you modifying the frame times being passed in to be wall times?

Reply:
Because I need build end time in millisecondsSinceEpoch to compare with the build end time from dart side.

Reply:
Why not just use unix wall time to start with then?

Reply:
The reason I do it this way instead of using wall time from the beginning is that it is easier to implement, the timing source is mixing but the accuracy is acceptable for my plugin, fixing it still outputs the same result for me.
On the other hand, adding code to record wall time is more complicated, harder to implement, and may affect other modules.
If you still think original wall time is needed, I will add the code.

Reply:
Understood that the timing accuracy is acceptable for your plugin but I'm asking you to change this PR so that it uses a single clock source and avoids doing iffy modifications to the time. It's important that our public APIs are extremely precise, consistent, and accurate.

Reply:
I added code to record wall time.